accord-core/src/main/java/accord/utils/Invariants.java

```diff
@@ -38,9 +38,12 @@ public enum ParanoiaCostFactor
         LOW, HIGH
     }
 
-    private static final int PARANOIA_COMPUTE = Paranoia.valueOf(System.getProperty("accord.paranoia.cpu", "NONE").toUpperCase()).ordinal();
-    private static final int PARANOIA_MEMORY = Paranoia.valueOf(System.getProperty("accord.paranoia.memory", "NONE").toUpperCase()).ordinal();
-    private static final int PARANOIA_FACTOR = ParanoiaCostFactor.valueOf(System.getProperty("accord.paranoia.costfactor", "LOW").toUpperCase()).ordinal();
+    public static final String KEY_PARANOIA_CPU = "accord.paranoia.cpu";
```

Comment:
in C* I put these into `CassandraRelativeProperties` so check styles doesn't yell at us